backend/services/stock.js

```diff
@@ -0,0 +1,60 @@
+const request = require('request-promise');
+const Stock = require ('../models/stock');
+const errors = require('../helpers/errors');
+const apiUrl = process.env.ALPHAVANTAGE_URL;
+const apiKey = process.env.ALPHAVANTAGE_API_KEY;
+
+const getDailyPrice = async (symbol) => {
+    const dailyParams ={function: "TIME_SERIES_INTRADAY",symbol:symbol,interval:"5min",apikey:apiKey};
+    const options = {
+        uri:apiUrl,
+        qs:dailyParams
+    }
+    let response = await request.get(options);
+    response = JSON.parse(response);
+    if(response['Meta Data']){
+        return response['Time Series (5min)'];
+    }else{
+        console.log("Error in daily Data")
+    }
+
+}
+
+const getMontlyPrice = async (symbol) => {
```

Comment:
typo, `getMontlyPrice` should be `getMonthlyPrice `

Reply:
ouch